tools/dev/run_dev_server.sh

```diff
@@ -666,7 +666,7 @@ case $CONFIG in
       CONTEXTIONARY_URL=localhost:9999 \
       AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED=true \
       DEFAULT_VECTORIZER_MODULE=text2vec-contextionary \
-      GOOGLE_CLOUD_PROJECT=project-id \
+      GOOGLE_CLOUD_PROJECT=voort-200517 \
```

Comment:
can you bring back the original setting? the one that you pushed is the name of your own gcp project

Reply:
thanks for catching that, done